examples/number_counts/number_counts_rich_values.ini

```diff
@@ -0,0 +1,26 @@
+; Parameters and data in CosmoSIS are organized into sections
+; so we can easily see what they mean.
+; There is only one section in this case, called cosmological_parameters
+[cosmological_parameters]
+
+; The following parameters are set, but not varied.
+;
+; omega_c is not set, because the consistency module sets it to omega_m - omega_b
+omega_m = 0.26
+sigma8_input = 0.8
+omega_k = 0.0
+omega_b = 0.0448
+tau = 0.08
+n_s = 0.963
+A_s = 2.1e-9
+h0 = 0.71
+w = -1.0
+wa = 0.0
+[firecrown_number_counts]
+cluster_counts_richness_proxy_mu_p0 = -5.0 1.0 5.0
+cluster_counts_richness_proxy_mu_p1 = -5.0 1.0 5.0
+cluster_counts_richness_proxy_mu_p2 = -5.0 1.0 5.0
+cluster_counts_richness_proxy_sigma_p0 = -5.0 1.0 5.0
+cluster_counts_richness_proxy_sigma_p1 = -5.0 1.0 5.0
+cluster_counts_richness_proxy_sigma_p2 = -5.0 1.0 5.0
```

Comment:
Do I understand correctly that in this example, you are using the counts to fit for the mass-richness relation parameters, for a fixed cosmology? Maybe comment this at the top of the file?

Reply:
Yes, but this is meant to be just an example file. If the user wants to analyze any of the cosmological parameters, they just need to change the parameter ranges. I'm going to write that on the file.